src/pages/index.js

```diff
@@ -2,29 +2,35 @@ import { getNextStaticProps } from '@faustjs/next';
 import Head from 'next/head';
 import React from 'react';
 import { client } from 'client';
-import { Posts, Header, LoadMore, Footer, Main } from 'components';
+import { FaArrowRight } from 'react-icons/fa';
+import {
+  Posts,
+  Header,
+  LoadMore,
+  Footer,
+  Main,
+  Button,
+  Heading,
+  CTA,
+  Testimonials,
+} from 'components';
 import appConfig from 'app.config';
 import usePagination from 'hooks/usePagination';
 
 export default function Page() {
   const { useQuery, usePosts } = client;
   const generalSettings = useQuery().generalSettings;
   const posts = usePosts({
-    first: appConfig.postsPerPage,
+    first: appConfig.homePagePostsCount,
     where: {
       categoryName: 'uncategorized',
     },
   });
-  const { data, fetchMore, isLoading } = usePagination(
-    (query, args) => {
-      const { nodes, pageInfo } = query.posts(args);
-      return {
-        nodes: Array.from(nodes),
-        pageInfo,
-      };
-    },
-    { nodes: posts?.nodes, pageInfo: posts?.pageInfo }
-  );
+  const testimonials = useQuery().testimonials();
+  const banner = {
```

Comment:
Yeah, good point. We do have to have `width` and `height` though, (and ideally `altText`) so a full `image` object still makes sense to me. It would be nice if `sourceUrl()` didn't have to be a function though. Could we test the type of `sourceUrl` in the `FeaturedImage` component and only call it if it's callable?